### ISSUE TITLE
[CI/CD]: Review coverage of accessibility checks in facility-locator end-to-end tests

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7358]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Applications **must** have thorough end-to-end tests that run in our continuous integration/continuous deployment (CI/CD) pipeline. These tests should include thorough axe checks. While auditing the Facility Locator with our VSA specialist, I noticed we have a new content box that appears when users select Urgent Care, Community Urgent Care from the search dropdown. I'd like the front-end team to update their e2e axe checks to include a branch that tests the first page with this dropdown present. Definition of done in acceptance criteria below. Screenshot attached at the bottom.

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## Environment
* `vets-website/src/applications/facility-locator/tests`
* `$ yarn test:e2e src/applications/facility-locator/tests`

## Acceptance Criteria

**Definition of done:**
- [ ] Front-end team member(s) have reviewed end-to-end tests for axe checks
- [ ] Add a flow to e2e test that causes the Community Care alert box to appear. Add a new axe check for that flow.
- [ ] FE team has consulted with accessibility specialist in cases where there are high numbers of modals, accordions, other hidden content that could slow down e2e test runs.
- [ ] No axe `violations` appear in the plugin console. These will break the CI/CD build.

## WCAG or Vendor Guidance (optional)

* [Custom axeCheck helper method](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js)
* [VSP guidance on writing end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/quality-assurance/e2e-testing)

---
![Screen Shot 2020-03-30 at 9 11 45 AM](https://user-images.githubusercontent.com/934879/77923156-a72a8100-7267-11ea-89b2-3462b2cd2b19.png)

**Comment by mmiddaugh** [comment]
Sending this back to the Icebox due to #2 in AC: 

> Add a flow to e2e test that causes the Community Care alert box to appear. Add a new axe check for that flow.

We know this workflow is challenging and likely to change in a future design iteration. See [#7239] for context and additional info.

**Comment by 1Copenut** [comment]
Thanks for the heads up @mmiddaugh . Can appreciate this is a complex flow and subject to change.

**Comment by mmiddaugh** [comment]
Can this be closed? We've changed that problematic workflow for Community care and expanded our test suite significantly.
